applications/configuration.md

```diff
@@ -0,0 +1,103 @@
+# Configuration
+
+When setting up configurables for an application we need to take into account several things.
+
+- Private and sensitive variables
+  - These can be stored in a private repo in GitLab
```

Comment:
None of this is a must, I just wanted it as a guide.  We store ours in a private repo on gitlab, which being behind a firewall and permissions based is as secure as using rattic, but you get source control, jenkins implementation etc.  rattic is just a password manager, isnt it?